File: pset6/mario/mario.py
from cs50 import get_int

def main():

    #Get pyramid's dimentions
	pyramidH = get_int("Pyrammid Height: ")


	x = 0
	y = pyramidH

    #Display Pyramid
	while True:
		spacer(y + 1)
		printer(x)
		print(" ", end="")
		printer(x)
		x += 1
		y -= 1
		print("")
		if x > pyramidH + 1:
		    break

#Print a set amount of ' '
def spacer(n):
	for x in range(n):
		print(" ", end="")

#Print a set amount of '#'
def printer(n):
	for x in range(n):
		print("#", end="")

main()

File: pset1/Mario/mario.c
#include <cs50.h>
#include <stdio.h>

void builder(int n)
    {
        for(int a = 0; a < n; a++){
            printf("#");
        };
    };
void spacer(int s)
    {
        for(int f = 0; f < s; f++){
            printf(" ");
        };
    };


int main(void)
{

    //Get pyramid's dimentions
    int pyramidH;

    do{
        pyramidH = get_int("Pyramid Height :");

    }while(pyramidH < 1 );


    //Display Pyramid

        int x = 0;
        int y = pyramidH;

        do{
            spacer(y);
            builder(x);
            printf(" ");
            builder(x);
            x++;
            y--;
            printf("\n");
        }while(x < pyramidH + 1);
}